src/uct/base/uct_cm.c

```diff
@@ -111,6 +111,23 @@ void uct_cm_ep_server_conn_notify_cb(uct_cm_base_ep_t *cep, ucs_status_t status)
     cep->server.notify_cb(&cep->super.super, cep->user_data, &notify_args);
 }
 
+int uct_cm_set_listener_backlog(const uct_listener_params_t *params,
+                                int default_value)
+{
+    int backlog;
+
+    if ((params->field_mask & UCT_LISTENER_PARAM_FIELD_BACKLOG) &&
+        (params->backlog == INT_MAX)) {
+        backlog = default_value;
+    } else if (params->field_mask & UCT_LISTENER_PARAM_FIELD_BACKLOG) {
+        backlog = params->backlog;
+    } else {
+        backlog = default_value;
+    }
```

Comment:
then it should be:
```
    if (params->field_mask & UCT_LISTENER_PARAM_FIELD_BACKLOG) {
        if (params->backlog > max_value) {
            ucs_diag(...);
        }
        backlog = ucs_min(params->backlog, max_value);
    } else {
        backlog = max_value;
    }

    if (backlog == 0) {
        ucs_error(...);
    }
```

Reply:
this means to cut the value for rdmacm as well.

Reply:
do you think rdmacm can support greater backlog than it's defined in `/proc/sys/net/rdma_ucm/max_backlog`?
My guessing is rdmacm behavior is similar to TCP, need to clarify this.